backend/src/main/java/gov/cdc/usds/simplereport/service/dataloader/TestOrderDeviceTypeDataLoader.java

```diff
@@ -1,41 +0,0 @@
-package gov.cdc.usds.simplereport.service.dataloader;
-
-import gov.cdc.usds.simplereport.db.model.DeviceType;
-import gov.cdc.usds.simplereport.db.model.TestOrder;
-import gov.cdc.usds.simplereport.db.repository.DeviceTypeRepository;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
-import java.util.concurrent.CompletableFuture;
-import java.util.stream.Collectors;
-import org.springframework.stereotype.Component;
-
-@Component
-public class TestOrderDeviceTypeDataLoader extends KeyedDataLoaderFactory<TestOrder, DeviceType> {
-  public static final String KEY = "testOrder[*].deviceType";
-
-  @Override
-  public String getKey() {
-    return KEY;
-  }
-
-  TestOrderDeviceTypeDataLoader(DeviceTypeRepository deviceTypeRepository) {
```

Comment:
Why are these other dataloaders being removed? Does the new GraphQL library just not use them?

Reply:
Nope, this all dead code.
this for example was replaced with
https://github.com/CDCgov/prime-simplereport/blob/d4c39da7936d1d07d0633dc3e18c7d1e242705f4/backend/src/main/java/gov/cdc/usds/simplereport/api/queue/ApiTestOrderDataResolver.java#L103-L107

https://github.com/CDCgov/prime-simplereport/blob/d4c39da7936d1d07d0633dc3e18c7d1e242705f4/backend/src/main/java/gov/cdc/usds/simplereport/api/queue/ApiTestOrderDataResolver.java#L44-L52